staging/src/k8s.io/legacy-cloud-providers/azure/clients/fileclient/azure_fileclient.go

```diff
@@ -46,27 +53,27 @@ func New(config *azclients.ClientConfig) *Client {
 }
 
 // CreateFileShare creates a file share
-func (c *Client) CreateFileShare(resourceGroupName, accountName, name string, protocol storage.EnabledProtocols, sizeGiB int) error {
-	result, err := c.GetFileShare(resourceGroupName, accountName, name)
+func (c *Client) CreateFileShare(resourceGroupName, accountName string, azureFileShare ShareInfo) error {
```

Comment:
should pass a pointer of `ShareOptions` in the function

Reply:
done